common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -355,10 +355,8 @@ def make_asset_id(course_id, asset_xml):
                     asset_md = AssetMetadata(asset_key)
                     asset_md.from_xml(asset)
                     all_assets.append(asset_md)
-        except OSError as os_exc:
-            msg = f'No {assets_filename} file is present with asset metadata.'
-            logging.error(f'Course import {course_id}: {msg}')
-            monitor_import_failure(course_id, 'Updating', msg, os_exc)
+        except OSError:
+            logging.error(f'Course import {course_id}: No {assets_filename} file present.')
```

Comment:
I would be confused why `monitor_import_failure` is not here if I ever saw this code in the future. I would recommend:
1. Adding a comment like `# Optional asset.xml file was not found.
2. Don't log an error if this is not an error. Either log nothing, or log info.

Lastly, are use sure this is the only case of OSError? Just a missing file? Is there any info in OSError that let's you know the file is missing, so we could still fail if it were a different OSError?

Reply:
Great feedback @robrap - I'll take that into account.

Reply:
Thanks.

Reply:
Addressed in the following commit. 
https://github.com/edx/edx-platform/commit/d8c9b4ec075e142d1ea06afe7ed0b2f0f389f2a2#diff-e96ae52c0c8d1f54e115acf553766ffb998294cd8ea548aa623a0f1b125f8216R360